garrysmod/lua/includes/modules/team.lua

```diff
@@ -170,8 +174,12 @@ end
 
 function GetColor( index )
 
-	if (!TeamInfo[index]) then return DefaultColor end
-	return table.Copy( TeamInfo[index].Color )
+	if ( !TeamInfo[index] ) then return DefaultColor end
+
+	local color = TeamInfo[index].Color
+	if ( color == nil ) then return nil end

```

Comment:
This should just return default color, or do not check this at all like the current version does.

Reply:
The current version uses table.Copy which does this check.